subt/tools/ignstate.py

```diff
@@ -121,8 +126,9 @@ def _read_world(cursor):
 
 
 def _parse_artifacts(world):
-    root = ET.fromstring(world[4:])
-    type_re = re.compile('^(backpack|rescue_randy|gas|vent|phone|artifact_origin|rope|helmet)')
+    start = world.find(b"<sdf")
+    root = ET.fromstring(world[start:])
```

Comment:
Why this change? The data should be xml and as such it should be parsed as xml.

Reply:
The original code fail when I run it on the "ver92tp1". There is the beginning of the "world": 
b"\x12\x90\x84\xc7\x01<sdf version='1.6'>\n
It seems that the number of ignored characters can be variable.

Reply:
Right. I forgot I didn't implement that properly. It should be fixed in #742. After we merge that, could please merge master to this PR so that we can remove the hack? Thanks.

Reply:
Done